src/memray/reporters/tree.py

```diff
@@ -232,6 +232,7 @@ class TreeApp(App[None]):
         Binding(
             key="e", action="expand_linear_group", description="Expand linear group"
         ),
+        [Binding("ctrl+z", "suspend_process")],
```

Comment:
Hm. _Technically_ this should use the `stty` `susp` binding, not hardcode `ctrl+z`... though to be fair I imagine almost no one changes it, so maybe it's not worth the bother.